synapse/crypto/keyring.py

```diff
@@ -815,87 +814,67 @@ async def _fetch_keys(
 
         async def get_key(key_to_fetch_item: _FetchKeyRequest) -> None:
             server_name = key_to_fetch_item.server_name
-            key_ids = key_to_fetch_item.key_ids
```

Comment:
I did wonder if `key_ids` was even needed here now, but it seems to be used quite a bit.